devtools/conda-envs/test_env.yaml

```diff
@@ -1,20 +1,24 @@
 name: test
 channels:
-
   - conda-forge
-
   - defaults
 dependencies:
-    # Base depends
+  # Base depends
   - python
   - pip
+  - jax
+  - openmm
+  - tqdm
+  - loguru
+  - openmm
+  - openmmtools
 
     # Testing
   - pytest
   - pytest-cov
   - codecov
 
     # Pip-only installs
-  #- pip:
+  - pip:
+     - h5py
```

Comment:
h5py is conda installable.  might want to move this up.

Reply:
thanks, moved it to the conda installs